tests/integration/helpers.py

```diff
@@ -80,22 +87,29 @@ async def get_unit_cidrs(model: Model, app_name: str, unit_num: int) -> List[str
     return list(sorted(local_cidrs))
 
 
-async def get_nodes(k8s):
-    """Return Node list
+async def get_rsc(k8s, resource, namespace=None, labels=None):
+    """Return Pod list
 
     Args:
         k8s: any k8s unit
+        resource: string resource type
+        namespace: string namespace
+        labels: dict of labels
 
     Returns:
-        list of nodes
+        list of pods
```

Comment:
nit: Improve the docstring. The method retrieves any k8s resource and returns a list of the specified resource.